governance/polls/MIP9 Community Greenlight Poll - U-DROP - July 19, 2021.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - Untangled Finance DROP (U-DROP) - July 19, 2021
```

Comment:
In fact these are the wrong way around as well. Template is $asset_code ($asset_name)